definitions/npm/underscore.string_v3.x.x/flow_all/underscore.string_v3.x.x.js

```diff
@@ -0,0 +1,87 @@
+declare module "underscore.string" {
+
+  declare type $npm$wrap$options = {
+    width?: number,
+    seperator?: string,
+    trailingSpaces?: boolean,
+    cut?: boolean,
+    preserveSpaces?: boolean,
+  };
+  declare type $npm$pad$type = 'left' | 'right' | 'both';
+
+  declare class S {
+    // TODO: fix it
+    static chain(string: string): any;
```

Comment:
No, you are right. Like I mentioned before, I couldn't come up with a solution that worked properly. I will try to implement it the way you suggested. Thanks 👍